lib/src/authentication-client.ts

```diff
@@ -508,7 +508,7 @@ export class AsgardeoAuthClient<T> {
         const stateParam = url.searchParams.get("state");
         const error = Boolean(url.searchParams.get("error"));
 
-        return stateParam && stateParam === SIGN_OUT_SUCCESS_PARAM && !error;
+        return stateParam ? stateParam === SIGN_OUT_SUCCESS_PARAM && !error: false;
```

Comment:
IMO, this is more readable. WDYT?
```suggestion
        return stateParam
            ? stateParam === SIGN_OUT_SUCCESS_PARAM && !error
            : false;
```